lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -16,62 +18,63 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
     final texts = context.texts();
+    return FutureBuilder<Config>(
+      future: _fetchConfig(),
+      builder: (BuildContext context, AsyncSnapshot<Config> snapshot) {
+        if (snapshot.hasData) {
+          final blockExplorer = snapshot.data!.defaultMempoolUrl;
+          if (paymentMinutiae.status == sdk.PaymentStatus.Complete) {
+            return Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                RichText(
+                  text: TextSpan(
+                    style: themeData.dialogTheme.contentTextStyle,
+                    text: texts.payment_details_dialog_closed_channel_local_wallet,
+                  ),
+                ),
+                if (paymentMinutiae.paymentType == sdk.PaymentType.ClosedChannel &&
+                    paymentMinutiae.fundingTxid != null) ...[
```

Comment:
What is the `paymentMinutiae.fundingTxid != null` check for?

Reply:
Added the wrong variable, fixed